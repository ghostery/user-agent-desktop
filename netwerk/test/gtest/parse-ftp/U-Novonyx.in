ftp> open pandora.anglistik.uni-mainz.de
220-  Novonyx FTP Server for NetWare, v0.51 (9-16-98)
220 Service Ready
Name (pandora.anglistik.uni-mainz.de:cyp): 
331 User name okay, need password
230 User logged in, proceed
Remote system type is UNIX.
Using binary mode to transfer files.
ftp> cd login
250 Directory successfully changed
ftp> ls
200 PORT Command OK
150 File status OK, about to open data connection
total 123
-rw-rw-rw-  1 root  sysadmin 25775 Sep 29 19:45 NOD$UPD.BAT
-r--r--r--  1 root  sysadmin 2860 Feb  6 22:03 AX_RUN.OVL
-r--r--r--  1 root  sysadmin 4965 Feb  6 22:03 B5DV_AIO.OVL
-r--r--r--  1 root  sysadmin 7843 Feb  6 22:03 B5DV_RUN.OVL
-r--r--r--  1 root  sysadmin 4965 Feb  6 22:03 B5ET_AIO.OVL
-r--r--r--  1 root  sysadmin 5589 Feb  6 22:03 B5ET_RUN.OVL
-r--r--r--  1 root  sysadmin 4965 Feb  6 22:03 B5KC_AIO.OVL
-r--r--r--  1 root  sysadmin 5042 Feb  6 22:03 B5KC_RUN.OVL
-r--r--r--  1 root  sysadmin 4965 Feb  6 22:03 BIG5_AIO.OVL
-r--r--r--  1 root  sysadmin 3710 Feb  6 22:03 BIG5_RUN.OVL
-r--r--r--  1 root  sysadmin 4965 Feb  6 22:03 CMPQ_AIO.OVL
-r--r--r--  1 root  sysadmin 2815 Feb  6 22:03 CMPQ_RUN.OVL
-r-xr-xr-x  1 root  sysadmin 210009 Feb  6 22:03 CX.EXE
-r--r--r--  1 root  sysadmin 4965 Feb  6 22:03 DOSV_AIO.OVL
-r--r--r--  1 root  sysadmin 3825 Feb  6 22:03 DOSV_RUN.OVL
-r--r--r--  1 root  sysadmin 16264 Feb  6 22:03 ETHER.RPL
-r--r--r--  1 root  sysadmin 12133 Feb  6 22:03 F1ETH.RPL
-r--r--r--  1 root  sysadmin 5141 Feb  6 22:03 FMR_AIO.OVL
-r--r--r--  1 root  sysadmin 4747 Feb  6 22:03 FMR_RUN.OVL
-r--r--r--  1 root  sysadmin 4965 Feb  6 22:03 GBCD_AIO.OVL
-r--r--r--  1 root  sysadmin 3857 Feb  6 22:03 GBCD_RUN.OVL
-r--r--r--  1 root  sysadmin 4965 Feb  6 22:03 GBET_AIO.OVL
-r--r--r--  1 root  sysadmin 5545 Feb  6 22:03 GBET_RUN.OVL
-r--r--r--  1 root  sysadmin 4965 Feb  6 22:03 GBLX_AIO.OVL
-r--r--r--  1 root  sysadmin 3913 Feb  6 22:03 GBLX_RUN.OVL
-r--r--r--  1 root  sysadmin 4965 Feb  6 22:03 GBPD_AIO.OVL
-r--r--r--  1 root  sysadmin 8019 Feb  6 22:03 GBPD_RUN.OVL
-r--r--r--  1 root  sysadmin 4965 Feb  6 22:03 GBSP_AIO.OVL
-r--r--r--  1 root  sysadmin 8141 Feb  6 22:03 GBSP_RUN.OVL
-r--r--r--  1 root  sysadmin 4965 Feb  6 22:03 GBTW_AIO.OVL
-r--r--r--  1 root  sysadmin 4152 Feb  6 22:03 GBTW_RUN.OVL
-r--r--r--  1 root  sysadmin 4965 Feb  6 22:03 GBUC_AIO.OVL
-r--r--r--  1 root  sysadmin 4033 Feb  6 22:03 GBUC_RUN.OVL
-r--r--r--  1 root  sysadmin 4965 Feb  6 22:03 GBWM_AIO.OVL
-r--r--r--  1 root  sysadmin 3710 Feb  6 22:03 GBWM_RUN.OVL
-r--r--r--  1 root  sysadmin 4965 Feb  6 22:03 GBXJ_AIO.OVL
-r--r--r--  1 root  sysadmin 3848 Feb  6 22:03 GBXJ_RUN.OVL
-r--r--r--  1 root  sysadmin 4965 Feb  6 22:03 GB_AIO.OVL
-r--r--r--  1 root  sysadmin 3710 Feb  6 22:03 GB_RUN.OVL
-r--r--r--  1 root  sysadmin 4965 Feb  6 22:03 IBM_AIO.OVL
-r--r--r--  1 root  sysadmin 2815 Feb  6 22:03 IBM_RUN.OVL
-r--r--r--  1 root  sysadmin 4965 Feb  6 22:03 J31_AIO.OVL
-r--r--r--  1 root  sysadmin 5240 Feb  6 22:03 J31_RUN.OVL
-r--r--r--  1 root  sysadmin 4965 Feb  6 22:03 KOR_AIO.OVL
-r--r--r--  1 root  sysadmin 3710 Feb  6 22:03 KOR_RUN.OVL
-r--r--r--  1 root  sysadmin 4965 Feb  6 22:03 KSC_AIO.OVL
-r--r--r--  1 root  sysadmin 3710 Feb  6 22:03 KSC_RUN.OVL
-r-xr-xr-x  1 root  sysadmin 311171 Apr  4 14:19 LOGIN.EXE
-r--r--r--  1 root  sysadmin 5167 Feb  6 22:03 PC98_AIO.OVL
-r--r--r--  1 root  sysadmin 7152 Feb  6 22:03 PC98_RUN.OVL
-r--r--r--  1 root  sysadmin 10991 Feb  6 22:03 PCN2L.RPL
-r--r--r--  1 root  sysadmin 4965 Feb  6 22:03 PS55_AIO.OVL
-r--r--r--  1 root  sysadmin 3616 Feb  6 22:03 PS55_RUN.OVL
-r--r--r--  1 root  sysadmin 8074 Feb  6 22:03 RBOOT.RPL
-r--r--r--  1 root  sysadmin 9170 Feb  6 22:03 TEXTUTIL.IDX
-r--r--r--  1 root  sysadmin 18433 Feb  6 22:03 TOKEN.RPL
-r-xr-xr-x  1 root  sysadmin 43549 Feb  6 22:03 TYPEMSG.EXE
-r--r--r--  1 root  sysadmin 27 Feb  6 22:03 ATTACH.BAT
-rwxrwxrwx  1 root  sysadmin 439913 Feb  6 23:33 NLIST.EXE
-r--r--r--  1 root  sysadmin 307 Feb  6 22:03 MENU_X.BAT
-rwxrwxrwx  1 root  sysadmin 269247 Feb  6 23:33 MAP.EXE
-r--r--r--  1 root  sysadmin 4965 Feb  6 22:03 AX_AIO.OVL
-r-xr-xr-x  1 root  sysadmin 111663 Apr 16 12:20 LOGIN3X.EXE
-rw-rw-rw-  1 root  sysadmin 38 Apr 18 15:19 login.nb
-r-xr-xr-x  1 root  sysadmin 392528 Apr 20 04:02 NPRINTER.EXE
-r-xr-xr-x  1 root  sysadmin 311171 Feb  9 16:29 LOGIN4X.EXE
-rw-rw-rw-  1 root  sysadmin 2102 Feb  8 13:18 lpt$cap.bat
-rw-rw-rw-  1 root  sysadmin 20614 Mar 13 13:42 nod$log.bat
-rw-rw-rw-  1 root  sysadmin 20450 Mar 12 10:36 nod$log.~ba
drwxrwxrwx  1 root  sysadmin      0 Mar  8 13:30 NLS
drwxrwxrwx  1 root  sysadmin      0 Mar  8 13:30 OS2
drwxrwxrwx  1 root  sysadmin      0 Apr  9 16:37 NOD$LOG.DRV
drwxrwxrwx  1 root  sysadmin      0 Apr  9 16:36 LOCAL
drwxrwxrwx  1 root  sysadmin      0 Mar 16 14:25 local.nt4
226 Requested file action okay, completed
ftp> close
221 Service closing connection, user logged out
ftp>
