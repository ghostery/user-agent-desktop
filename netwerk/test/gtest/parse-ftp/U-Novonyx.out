<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->

09-29-2001  19:45:00                25775 NOD$UPD.BAT
02-06-2002  22:03:00                 2860 AX_RUN.OVL
02-06-2002  22:03:00                 4965 B5DV_AIO.OVL
02-06-2002  22:03:00                 7843 B5DV_RUN.OVL
02-06-2002  22:03:00                 4965 B5ET_AIO.OVL
02-06-2002  22:03:00                 5589 B5ET_RUN.OVL
02-06-2002  22:03:00                 4965 B5KC_AIO.OVL
02-06-2002  22:03:00                 5042 B5KC_RUN.OVL
02-06-2002  22:03:00                 4965 BIG5_AIO.OVL
02-06-2002  22:03:00                 3710 BIG5_RUN.OVL
02-06-2002  22:03:00                 4965 CMPQ_AIO.OVL
02-06-2002  22:03:00                 2815 CMPQ_RUN.OVL
02-06-2002  22:03:00               210009 CX.EXE
02-06-2002  22:03:00                 4965 DOSV_AIO.OVL
02-06-2002  22:03:00                 3825 DOSV_RUN.OVL
02-06-2002  22:03:00                16264 ETHER.RPL
02-06-2002  22:03:00                12133 F1ETH.RPL
02-06-2002  22:03:00                 5141 FMR_AIO.OVL
02-06-2002  22:03:00                 4747 FMR_RUN.OVL
02-06-2002  22:03:00                 4965 GBCD_AIO.OVL
02-06-2002  22:03:00                 3857 GBCD_RUN.OVL
02-06-2002  22:03:00                 4965 GBET_AIO.OVL
02-06-2002  22:03:00                 5545 GBET_RUN.OVL
02-06-2002  22:03:00                 4965 GBLX_AIO.OVL
02-06-2002  22:03:00                 3913 GBLX_RUN.OVL
02-06-2002  22:03:00                 4965 GBPD_AIO.OVL
02-06-2002  22:03:00                 8019 GBPD_RUN.OVL
02-06-2002  22:03:00                 4965 GBSP_AIO.OVL
02-06-2002  22:03:00                 8141 GBSP_RUN.OVL
02-06-2002  22:03:00                 4965 GBTW_AIO.OVL
02-06-2002  22:03:00                 4152 GBTW_RUN.OVL
02-06-2002  22:03:00                 4965 GBUC_AIO.OVL
02-06-2002  22:03:00                 4033 GBUC_RUN.OVL
02-06-2002  22:03:00                 4965 GBWM_AIO.OVL
02-06-2002  22:03:00                 3710 GBWM_RUN.OVL
02-06-2002  22:03:00                 4965 GBXJ_AIO.OVL
02-06-2002  22:03:00                 3848 GBXJ_RUN.OVL
02-06-2002  22:03:00                 4965 GB_AIO.OVL
02-06-2002  22:03:00                 3710 GB_RUN.OVL
02-06-2002  22:03:00                 4965 IBM_AIO.OVL
02-06-2002  22:03:00                 2815 IBM_RUN.OVL
02-06-2002  22:03:00                 4965 J31_AIO.OVL
02-06-2002  22:03:00                 5240 J31_RUN.OVL
02-06-2002  22:03:00                 4965 KOR_AIO.OVL
02-06-2002  22:03:00                 3710 KOR_RUN.OVL
02-06-2002  22:03:00                 4965 KSC_AIO.OVL
02-06-2002  22:03:00                 3710 KSC_RUN.OVL
04-04-2002  14:19:00               311171 LOGIN.EXE
02-06-2002  22:03:00                 5167 PC98_AIO.OVL
02-06-2002  22:03:00                 7152 PC98_RUN.OVL
02-06-2002  22:03:00                10991 PCN2L.RPL
02-06-2002  22:03:00                 4965 PS55_AIO.OVL
02-06-2002  22:03:00                 3616 PS55_RUN.OVL
02-06-2002  22:03:00                 8074 RBOOT.RPL
02-06-2002  22:03:00                 9170 TEXTUTIL.IDX
02-06-2002  22:03:00                18433 TOKEN.RPL
02-06-2002  22:03:00                43549 TYPEMSG.EXE
02-06-2002  22:03:00                   27 ATTACH.BAT
02-06-2002  23:33:00               439913 NLIST.EXE
02-06-2002  22:03:00                  307 MENU_X.BAT
02-06-2002  23:33:00               269247 MAP.EXE
02-06-2002  22:03:00                 4965 AX_AIO.OVL
04-16-2002  12:20:00               111663 LOGIN3X.EXE
04-18-2002  15:19:00                   38 login.nb
04-20-2002  04:02:00               392528 NPRINTER.EXE
02-09-2002  16:29:00               311171 LOGIN4X.EXE
02-08-2002  13:18:00                 2102 lpt$cap.bat
03-13-2002  13:42:00                20614 nod$log.bat
03-12-2002  10:36:00                20450 nod$log.~ba
03-08-2002  13:30:00       <DIR>          NLS
03-08-2002  13:30:00       <DIR>          OS2
04-09-2002  16:37:00       <DIR>          NOD$LOG.DRV
04-09-2002  16:36:00       <DIR>          LOCAL
03-16-2002  14:25:00       <DIR>          local.nt4
